code/modules/mapping/random_rooms.dm

```diff
@@ -1479,3 +1479,139 @@
 	centerspawner = FALSE
 	template_height = 11
 	template_width = 5
+
+/datum/map_template/random_room/skwh_rdm158
+	name = "Large Abandoned Virology"
+	room_id = "skwh_rdm158_abandonedviro"
+	mappath = "_maps/RandomRooms/10x10/skwh_rdm158_abandonedviro.dmm"
+	centerspawner = FALSE
+	template_height = 10
+	template_width = 10
+	weight = 2
+	stock = 3
+
+/datum/map_template/random_room/skwh_rdm157
+	name = "Slime Storage"
+	room_id = "skwh_rdm157_slimestorage"
+	mappath = "_maps/RandomRooms/10x5/skwh_rdm157_slimestorage.dmm"
+	centerspawner = FALSE
+	template_height = 5
+	template_width = 10
+	weight = 3
+
+/datum/map_template/random_room/skwh_rdm159
+	name = "Monkey Testing Room"
+	room_id = "skwh_rdm159_MONKETESTSUBJECT"
+	mappath = "_maps/RandomRooms/10x10/skwh_rdm159_MONKETESTSUBJECT.dmm"
+	centerspawner = FALSE
+	template_height = 10
+	template_width = 10
+	weight = 1
+	stock = 5
+
+/datum/map_template/random_room/skwh_rdm139
+	name = "SAW GBS Maint Game Room"
+	room_id = "skwh_rdm139_GBSsawgameroom"
+	mappath = "_maps/RandomRooms/5x3/skwh_rdm139_GBSsawgameroom.dmm"
+	centerspawner = FALSE
+	template_height = 3
+	template_width = 5
+	weight = 2
+	stock = 1
+
+/datum/map_template/random_room/skwh_rdm160
+	name = "Green Viney Riches"
+	room_id = "skwh_rdm160_greenvineyriches"
```

Comment:
Contains far too many virus samples.
The space cash there has 0 value.
Contains security IDs and antagonist gear.
That many kudzu seeds are going to lead to issues.

Reply:
huh i thought i put more fresh blood then dried blood in this room, and i'll remove the dead bodies that have the sec ids and antagonist gear

Reply:
also just question when you mean that space cash has 0 value, do you mean it should spawn with 0 value or does it just spawn with 0 value? cause in the paper scrap i put in the room with what i wrote in it, i intended the space cash in the room to have 0 value but forgot to change many of the space cash/credits to 0 value, cause i intended the money to be "counterfitted nanotrasen credits" or something like that, sorry for being a bit confused by what you mean by "The space cash has 0 value"

Reply:
wait no now i know what you meant you mean the regular space cash has 0 value aight removing those sorry for the confusion